scriptmodules/emulators/amiberry.sh

```diff
@@ -106,12 +106,15 @@ function configure_amiberry() {
 
     # move config / save folders to $md_conf_root/amiga/amiberry
     local dir
-    for dir in conf savestates screenshots; do
+    for dir in conf nvram savestates screenshots; do
         moveConfigDir "$md_inst/$dir" "$md_conf_root/amiga/amiberry/$dir"
     done
 
-    # symlink cd32.nvr from $md_inst/data to "$md_conf_root/amiga/amiberry
-    moveConfigFile "$md_inst/data/cd32.nvr" "$md_conf_root/amiga/amiberry/cd32.nvr"
+    # check for cd32.nvr and move it to $md_conf_root/amiga/amiberry/nvram
+    if [[ -f "$md_inst/data/cd32.nvr" ]]; then
+	mv "$md_inst/data/cd32.nvr" "$md_conf_root/amiga/amiberry/nvram/"
```

Comment:
Sorry I just saw. I think this should be `mv $md_conf_root/amiga/amiberry/cd32.nvr "$md_conf_root/amiga/amiberry/nvram/"` ? To move the old location to the new. The path `$md_inst/data/cd32.nvr" ` is a symlink.

Reply:
Silly me! The rookie mistake with symlink 🤦‍♂️ Fixing it...

Reply:
The `if [[ -f` should use that path too.

Reply:
Looks good. Thanks.